packages/bpk-component-breadcrumb/src/BpkBreadcrumb.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './BpkBreadcrumb.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkBreadcrumb = props => {
+  const { children } = props;
+
+  const className = getClassName('bpk-breadcrumb');
+
+  return <div className={className}>{children}</div>;
```

Comment:
I propose that the html for breadcrumbs follows [this example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/breadcrumb/index.html). So in this case, it should be `<nav><ol /></nav>` for the `BpkBreadcumb` component:
```
  <nav aria-label="Breadcrumb" class="breadcrumb">
    <ol>
      <li>
        <a href="../../">
          WAI-ARIA Authoring Practices 1.1
        </a>
      </li>
      <li>
        <a href="../../#aria_ex">
          Design Patterns
        </a>
      </li>
      <li>
        <a href="../../#breadcrumb">
          Breadcrumb Pattern
        </a>
      </li>
      <li>
        <a href="./index.html" aria-current="page">
          Breadcrumb Example
        </a>
      </li>
    </ol>
  </nav>
```